src/seedu/addressbook/commands/InfoCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.commands;
+
+public class InfoCommand extends Command{
```

Comment:
It is better to have a comment explaining the responsibility of this class to make it more clear.